### ISSUE TITLE
0.8.0 -> 0.9.0 Upgrade issue with pretrained resnet

### ISSUE THREAD

**Issue by silvanheller** [bytedeco/javacpp-presets#274]
Executing the pre-trained ResNet converted to tensorflow ([repo](https://github.com/ry/tensorflow-resnet)
Gives the following error:
`
Exception in thread "main" java.lang.RuntimeException: NodeDef mentions attr 'T' not in Op output:float; attr=ksize:list(int),min=4; attr=strides:list(int),min=4; attr=padding:string,allowed=["SAME", "VALID"]; attr=data_format:string,default="NHWC",allowed=["NHWC", "NCHW"]>; NodeDef: scale2/MaxPool = MaxPoolT=DT_FLOAT, data_format="NHWC", ksize=[1, 3, 3, 1], padding="SAME", strides=[1, 2, 2, 1], _device="/device:CPU:0"
`

The models runs good in tensorflow-python. I have also frozen the graph into a full .pb-file which does not help with the issue.

Steps to reproduce this issue:

        tensorflow.GraphDef def = new tensorflow.GraphDef();
        tensorflow.ReadBinaryProto(tensorflow.Env.Default(),
                model, def);
        tensorflow.Status s = session.Create(def);
        if (!s.ok()) {
            LOGGER.error("Error while loading graph");
            throw new RuntimeException(s.error_message().getString());

where model is the model which can be found on the linked github page. See [resnet-issue](https://github.com/ry/tensorflow-resnet/issues/2) and [tf-issue](https://github.com/tensorflow/tensorflow/issues/1528)

It might be because the resnet depends on commit https://github.com/tensorflow/tensorflow/commit/cf7ce8a7879b6a7ba90441724ea3f8353917a515 as mentioned in ry's github README but that should not be the case since the javacpp-presets are updated on 0.9.0

Oh, and if someone has made it this far - the readme.md is wrong. 0.9.0 is up on maven

**Comment by saudet** [comment]
Maybe the following guide might help?
https://medium.com/google-cloud/how-to-invoke-a-trained-tensorflow-model-from-java-programs-27ed5f4f502d#.rqdl9d41s

**Comment by silvanheller** [comment]
Thanks for the quick response. I'm aware of that article - my code is almost exactly the same since I've largely copied it from the article. This works well with the pretrained VGG16 Net.
My error here seems to be a versioning issue since it comes up both as an issue in the main tensorflow-repo and in the other repo I've linked. I can upload the metagraph if you want to reproduce the issue and can't get the torrent.

**Comment by saudet** [comment]
So it's a bug in TensorFlow itself? Not sure I understand what you're asking about here...

**Comment by saudet** [comment]
From what I understand, it's a bug that was fixed in v0.10.0rc0, so let's mark this as a duplicate of #266
